packages/server/src/lib/fetchTickerData.ts

```diff
@@ -33,25 +33,31 @@ const parse = (json: any): Promise<TickerData> => {
     }
 };
 
+const getTickerDataForTickerTypeFetcher = (type: TickerCountType) => () => {
+    return fetch(tickerUrl(type), {
+        timeout: 1000 * 20,
+    })
+        .then(response => checkForErrors(response))
+        .then(response => response.json())
+        .then(parse);
+};
+
+const cachedTickerFetchers = {
+    [TickerCountType.people]: cacheAsync(
+        getTickerDataForTickerTypeFetcher(TickerCountType.people),
+        {
+            ttlSec: 60,
+        },
+    ),
+    [TickerCountType.money]: cacheAsync(getTickerDataForTickerTypeFetcher(TickerCountType.money), {
+        ttlSec: 60,
+    }),
+};
+
```

Comment:
This is the only place that required a sizeable change. Before `fetchTickerDataCached` was actually creating a new cached function on every invocation. However, because the cached function checked the cache based on the cache key, it would still read the same value from the cache instead of making the fetch request. Now we just create the cached functions once and reuse them every time.